roles/network_plugin/calico/templates/cni-calico.conflist.j2

```diff
@@ -28,6 +28,15 @@
         "subnet": "usePodCidr"
       },
 {% endif %}
+{% if calico_feature_control|length > 0 %}
```

Comment:
This will fail if `calico_feature_control` suddenly is not defined. I would do:

`{% if calico_feature_control is defined and if calico_feature_control|length > 0 %}`